.changes/unreleased/operator-Changed-20250109-091351.yaml

```diff
@@ -1,10 +1,10 @@
 project: operator
 kind: Changed
 body: |
-  For any user that is mirroring configurator image and changes entrypoint or wraps configurator with additional 
-    script the following constraint need to be meet:
-    * use the alternative configuration parameter by specifying `additionalCmdFlags` in Redpanda operator helm chart
-      * to change the container repository set `--configurator-base-image=my.repo.com/configurator`
-      * to change the container tag set `--configurator-tag=XYZ`
+  For any user that is mirroring configurator image (air-gapped environment) and changes entrypoint
+    or wraps configurator with additional script the following constraint need to be meet:
+    * set the following flags
+      * to change the container repository set `--configurator-base-image=my.repo.com/configurator` flag
```

Comment:
I don't think this needs to be specified as this behavior did not change? Or is this meant to call out that the default repository/image has changed?